web-src/src/common/components/textfield.vue

```diff
@@ -191,8 +192,15 @@ export default {
   }
 }
 
-function getValidByTypeError(value, type, min, max, max_length) {
+function getValidByTypeError(value, type, min, max, max_length, regex) {
   if (type === 'text') {
+    if (regex) {
+        let regexPattern = new RegExp(regex.pattern);
+        let matchFound = regexPattern.test(value);
+        if (!matchFound) {
+          return regex.description
```

Comment:
Since description can be empty, please provide a fallback value

Reply:
When/if description is left empty, `regex.description` is the regex pattern itself. The idea is to provide the end-user with a meaningful error message when it comes to regex validation but if, for some reason, you don't, shouldn't print the regex pattern is enough of a fallback value?
https://github.com/bugy/script-server/blob/8ade772bb7218237b7ae741f895b7d960987c076/src/model/parameter_config.py#L302

Reply:
The python code you are referencing is a backend validation, a user should never see it. It's just an additional safety net.